scripts/npm/link/index.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var path = require( 'path' );
+var fs = require( 'fs' );
+var componentsDir = path.join( __dirname, '..', '..', '..', 'components' );
+var packageContent = fs.readFileSync( path.join( __dirname, '..', '..', '..', 'package.json' ) );
+var packageJSON = JSON.parse( packageContent );
+var exec = require( 'child-process-promise' ).exec;
+
+function filterComponents( deps ) {
+  for ( var component in deps ) {
+    if ( /^cf-/.test( component ) ) {
```

Comment:
Yeah. This isn't the first time we've run into an issue of that sort. We could create a standard set of secondary prefixes, like `cf-theme-` and `cf-util-` for non-component things that we still want to be associated with CF? But I agree that it's not a blocker for this PR.